glad/documentation.py

```diff
@@ -0,0 +1,190 @@
+import re
+import glad.util
+from lxml import etree
+from glad.parse import DocumentationSet, SpecificationDocs, CommandDocs, xml_parse
+from glad.util import prefix, suffix, raw_text
+
+class OpenGLRefpages(SpecificationDocs):
+    DOCS_NAME = 'opengl_refpages'
+
+    URL = 'https://github.com/KhronosGroup/OpenGL-Refpages/archive/refs/heads/main.zip'
+    SPEC = 'gl'
+
+    def select(self, feature_set):
+        current_major = list(feature_set.info)[0].version.major
```

Comment:
When merging multiple specs this can contain multiple infos of all the merged versions.

Seems like this should work here:
```python
max(info.version.major for info in feature_set.info)
```